relion/protocols/protocol_reconstruct.py

```diff
@@ -123,7 +123,9 @@ def _getProgram(self, program='relion_reconstruct'):
         if self.numberOfMpi > 1:
             program += '_mpi'
         if self.numberOfThreads > 1 and not IS_V2:
-            program += '_openmp'
+            # TODO: comment josemiguel
+            pass #
```

Comment:
what is this?

Reply:
I don't know...I remember that you add previous line was to provide openmp support in the case of the reconstruct protocol... @rmarabini why did you change this?